docs/api/python/contrib/summary.md

```diff
@@ -0,0 +1,258 @@
+# Logging MXNet Data for Visualization in TensorBoard
```

Comment:
Do we want to call it summary? Should the name be something that explicitly points to tensorboard?